pyproject.toml

```diff
@@ -36,73 +36,16 @@ dependencies = [
     "torchmetrics>=1.4.0.post0,<2.0.0",
     "wandb>=0.18.0,<1.0.0",
 ]
+[build-system]
+requires = ["setuptools >= 72.1.0", "wheel"]
+build-backend = "setuptools.build_meta"
 
 [project.urls]
 homepage = "https://github.com/CUNY-CL/udtube"
 
-[build-system]
-requires = ["poetry-core>=2.0.0"]
-build-backend = "poetry.core.masonry.api"
-
-[tool.poetry]
-name = "udtube"
-version = "0.1.3"
-description = "Neural morphological analysis"
-readme = "README.md"
-license = "Apache-2.0"
-authors = [
-    "Daniel Yakubov <danielyak98@gmail.com>",
-    "Kyle Gorman <kylebgorman@gmail.com"
-]
-keywords = [
-    "computational linguistics",
-    "morphology",
-    "natural language processing",
-    "language",
-]
-classifiers = [
-    "Programming Language :: Python :: 3.9",
-    "Programming Language :: Python :: 3.10",
-    "Programming Language :: Python :: 3.11",
-    "Programming Language :: Python :: 3.12",
-    "Programming Language :: Python :: 3.13",
-    "Development Status :: 4 - Beta",
-    "Environment :: Console",
-    "License :: OSI Approved :: Apache Software License",
-    "Operating System :: OS Independent",
-    "Topic :: Text Processing :: Linguistic",
-]
+[tool.setuptools.packages.find]
 include = ["udtube*"]
-exclude = ["configs*",  "examples*", "scripts*" ]
-
-[tool.poetry.dependencies]
-jsonargparse = { version = "^4.32.0", extras = ["signatures"] }
-lightning = "^2.4.0"
-python = ">=3.9,<3.14"
-pyyaml = "^6.0"
-transformers = "^4.44.0"
-torch = "^2.4.0"
-torchmetrics = "^1.4.0.post0"
-wandb = "^0.18.0"
-
-[tool.poetry.group.dev]
-optional = true
-
-[tool.poetry.group.dev.dependencies]
-black = "^24.4.2"
-build = "^1.2.1"
-flake8 = "^7.1.0"
-twine = "^6.1.0"
-
-[tool.poetry.group.test]
-optional = true
-
-[tool.poetry.group.test.dependencies]
-parameterized = "^0.9.0"
-pytest = "^7.4.4"
-
-[tool.poetry.scripts]
```

Comment:
You need an equivalent of this line in whatever framework you choose to install the script. What I was doing before Poetry should be in the commit history but it's presumably parallel to [this](https://github.com/CUNY-CL/yoyodyne/blob/a9e00597ee787ed9109d4538798452982a6dfdd8/pyproject.toml#L52).

Reply:
Good catch. Added